guides/wide-ep-lws/manifests/gateway/gke-l7-regional-external-managed/kustomization.yaml

```diff
@@ -2,33 +2,9 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ../base
-patches:
-- path: gateway.yaml
-  target:
-    kind: Gateway
-- target:
-    kind: HTTPRoute
-  # Remove timeouts since GKE gateway doesn't support it.
-  # Use GCPBackendPolicy to configure timeouts on GKE.
-  patch: |-
-    - op: remove
-      path: /spec/rules/0/timeouts
-- target:
-    kind: HTTPRoute
-  # Adjust the backend ref on the httpRoute to the correct v1 apiVersion
-  patch: |-
-    - op: replace
-      path: /spec/rules/0/backendRefs/0/group
-      value: inference.networking.k8s.io
-- target:
-    kind: HTTPRoute
-  # Remove port value for the inference pool in the v1 apis
-  patch: |-
-    - op: remove
-      path: /spec/rules/0/backendRefs/0/port
+- ../../../../recipes/gateway/gke-l7-regional-external-managed
 labels:
 - pairs:
     app.kubernetes.io/instance: wide-ep
-    app.kubernetes.io/gateway: wide-ep-inference-gateway
+    app.kubernetes.io/gateway: llm-d-inference-gateway
```

Comment:
It looks like we keep the kustomization here just for these labels. I think these are for informational purpose and it's OK not having them. So we should be able to get rid of these and use the recipe directly.